stories/Containers/QRScan.stories.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { QRScan, QRScanProps, PhoneBox, qrImgURL1 } from '../../src';
+import { createStoryTitle } from '../Constants';
+import { Meta, Story } from '@storybook/react';
+
+import { MainTheme } from '../../src/Themes';
+
+export default {
+    title: createStoryTitle('QRScan'),
+    component: QRScan,
```

Comment:
Put the args here for duplicate args btw that way you don't need to redeclare everytime

Reply:
ok, will try and commit again...

Reply:
I tried but it doesn't work as expected. they don't work as default args. if I don't put them outside the default they don't appear on the component.

Reply:
They should work, have you looked at the other components that have implemented this (Look at Button)

Reply:
Ok, I made a separate object defaultArgs and used js destructuring. committed it.